lib/solaar/ui/window.py

```diff
@@ -578,16 +562,15 @@ def _update_receiver_panel(receiver, panel, buttons, full=False):
 	assert receiver
 
 	devices_count = len(receiver)
-	if receiver.max_devices > 1:
-		if devices_count == 0:
-			panel._count.set_markup(_UNIFYING_RECEIVER_TEXT[0] % receiver.max_devices)
-		else:
-			panel._count.set_markup(_UNIFYING_RECEIVER_TEXT[1] % (devices_count, receiver.max_devices))
-	else:
-		if devices_count == 0:
-			panel._count.set_markup(_NANO_RECEIVER_TEXT[0])
-		else:
-			panel._count.set_markup(_NANO_RECEIVER_TEXT[1])
+	
+	paired_text = gettext.gettext('No device paired.') if devices_count == 0 else gettext.ngettext('One paired device.', '%(devices_count)s paired devices.', devices_count) % { 'devices_count': devices_count }
+		
+	if(receiver.max_devices > 0):
+		paired_text += gettext.ngettext('\n\n<small>Up to %(max_devices_count)s device can be paired to this receiver.</small>', '\n\n<small>Up to %(max_devices_count)s devices can be paired to this receiver.</small>', receiver.max_devices) % { 'max_devices_count': receiver.max_devices }
```

Comment:
html is ok in translations:
HTML markup, however, is common enough that it’s probably ok to use in translatable strings. But please bear in mind that the GNU gettext tools don’t verify that the translations are well-formed HTML.
https://www.gnu.org/software/gettext/manual/gettext.html